library/item/item_rendering/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Item Rendering API"
+	moduleName = "item_rendering"
+	id = "quilt_item_rendering"
+	description = "An API for modifying and extending the rendering of items in inventories."
+	library = "item"
+	moduleDependencies {
+		core {
+			api("qsl_base")
+		}
+	}
+	injectedInterface("net/minecraft/class_1792") {
+		values = ["org/quiltmc/qsl/item/rendering/client/api/QuiltItemRenderingExtensions"]
```

Comment:
this should be ``org/quiltmc/qsl/item/rendering/api/client/QuiltItemRenderingExtensions`` instead of ``org/quiltmc/qsl/item/rendering/client/api/QuiltItemRenderingExtensions``